src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentContext.cs

```diff
@@ -5,6 +5,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
```

Comment:
Huh, what part of the additions brings this along?

Reply:
`LocateOwner` is an extension method in this namespace